### ISSUE TITLE
[A11y] List element on search page cannot have a direct child <br> element

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#9402]
### Accessibility issue on the search page

A list element cannot have a child line break element. We wanted to place the package tags on a new line, so we'll have to find another way to do this.

![image](https://user-images.githubusercontent.com/82980589/221028298-917d519f-baf6-480f-b899-5012c955624f.png)
